conf/fluo-deploy.props.example

```diff
@@ -54,6 +54,8 @@ default.instance.type = m3.large
 key.name = my_aws_key
 # Tags to add instances
 #instance.tags = key1:value1,key2:value2
+#Number of ephemeral drives to setup.  Do not set higher than supported by instance type.
+#ephemeral.num = 0
```

Comment:
In `[nodes]`, users can specify an instance type for each node (or `default` if they want to use the instance type declared by the `default.instance.type` property).  

However, the `ephemeral.num` property sets the number of ephemeral drives for the whole cluster which will cause problems if some nodes have 2 drives and others have 1 drive.  However, letting nodes have different numbers of drives for different nodes can make things complicated (especially if datanodes are not consistent).  Therefore, I would be fine if we just made the cluster use only one instance type for now and removed the instance type declaration in `[nodes]`.  It might be nice to remove the `ephemeral.num` property and just keep a map of instance types to number of ephemeral drives.  It will save time for users.